README.md

```diff
@@ -2,46 +2,101 @@
-![](https://img.shields.io/github/issues-pr-raw/appuio/seiso)
-![](https://img.shields.io/github/license/appuio/seiso)
+[![](https://img.shields.io/github/workflow/status/appuio/seiso/Build)](https://github.com/appuio/seiso/actions)
+[![](https://img.shields.io/github/v/release/appuio/seiso?include_prereleases)](https://github.com/appuio/seiso/releases)
+[![](https://img.shields.io/github/issues-raw/appuio/seiso)](https://github.com/appuio/seiso/issues)
+[![](https://img.shields.io/github/issues-pr-raw/appuio/seiso)](https://github.com/appuio/seiso/pulls)
+[![](https://img.shields.io/github/license/appuio/seiso)](https://github.com/appuio/seiso/blob/master/LICENSE)
 
-## General
+Inspired by Robert C. Martin's book, [Clean Code](https://www.investigatii.md/uploads/resurse/Clean_Code.pdf), foreword, page xx:
 
-Seiso is a client that is used to clean up container images in an image registry (currently only OpenShift image registries
-are supported).
+> Seisō (清掃), Japanese for “cleaning” (think “shine” in English):
+> Keep the workplace free of hanging wires, grease, scraps, and waste.
 
-The tool is intended to be used closely within an application lifecycle management (Review apps). It analyzes a git repository
-and compares the history with the target image registry, removing old and unused image tags according to customizable rules.
+## Clean up your container registry and Kubernetes resources
 
-The tool can also be used more aggressively by deleting unknown image tags too ("orphans"). See more usage examples below.
+*Seiso* is a CLI client that helps you clean up container resources.
 
-This tool is opinionated in the naming of image tags, as the image tags have to follow certain naming rules for this to work.
-E.g. stick to tagging with Git SHA value: `namespace/app:a3d0df2c5060b87650df6a94a0a9600510303003` or Git Tag: `namespace/app:v1.2.3`.
+* Unused images in your container registry (identified by Image Stream Tags
+  in an [Image Stream](https://blog.openshift.com/image-streams-faq/) in OpenShift)
 
-This helps to save space (e.g. your private registry with billed S3 storage) or quota threshold because obsolete images are
-being removed from the registry.
+* Superflous [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#understanding-configmaps-and-pods)s
+  in your Kubernetes cluster (e.g. generated by the Kustomize [configMapGenerator](
+  https://kubectl.docs.kubernetes.io/pages/reference/kustomize.html#configmapgenerator))
 
-The cleanup **runs in dry-mode by default**, only when the `--force` flag is specified, it will delete the actual image tags.
-This prevents accidental deletions when testing.
+* Superflous [Secret](https://kubernetes.io/docs/concepts/configuration/secret/)
+  objects in your Kubernetes cluster (e.g. generated by the Kustomize [secretGenerator](
+  https://kubectl.docs.kubernetes.io/pages/reference/kustomize.html#secretgenerator))
```

Comment:
I don't think we should document features that aren't even implemented. Please remove.